server/plugin/plugin.go

```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/mattermost/mattermost-plugin-apps/server/configurator"
 	"github.com/mattermost/mattermost-plugin-apps/server/constants"
 	myhttp "github.com/mattermost/mattermost-plugin-apps/server/http"
+	"github.com/mattermost/mattermost-plugin-apps/server/http/api"
```

Comment:
1/5 recommend package name either `restapi` or `httpapi` to avoid collisions and reveal the intention. This package name should be used rarely (`plugin.go` only?) so making it (even more?) verbose is ok.

Reply:
I like restapi for the package naming.  committed the change.